addon/ng2/utilities/get-dependent-files.ts

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+import * as fs from 'fs';
+import * as ts from 'typescript';
+import * as glob from 'glob';
+import * as path from 'path';
+
+/**
+ * Declare a MapType where key is a string and value is an array of type string
+ * Use for storing a file's path as key and its dependent module's relative path
+ */
+declare interface ModuleImport {
+  specifier: string;
+  pos: number;
+  end: number;
+};
+
+/**
+ * Create a SourceFile as defined by Typescript Compiler API.
+ * Generate a AST structure from a source file.
+ * 
+ * @param fileName source file for which AST is to be extracted
+ */
+export function createTsSourceFile(fileName: string): ts.SourceFile {
+  return ts.createSourceFile(fileName,
+                             fs.readFileSync(fileName, 'utf-8'),
+                             ts.ScriptTarget.ES6,
+                             true);
+}
+
+/**
+ * Traverse through ts.Node recursively and stores moduleSpecifiers in an array
+ * 
+ * @param {ts.Node} node: Typescript Node of whose AST is being traversed
+ * @param {string[]} moduleSpecifiers: array to store moduleSpecifiers if found.
+ * 
+ */
+function _getImportClauses(node: ts.Node): ModuleImport[] {
+  return node.getChildren()
```

Comment:
`node` will be a `SourceFile` so just use `SourceFile.statements` instead of `getChildren()`.